modules/operators-overview.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='operators-overview-{context}']
+= Operators in {product-title}
+
+In {product-title} version 4.0, all cluster functions are divided into a series
+of component Operators.
+
+Operators can manage other Operators within their component. This lends to a
+transparent model: a single Operator usually manages a single binary file.
+Operators also offer a more granual configuration experience. You configure each component by
+modifying its Operator instead of modifying a global configuration file.
+
+In version 4.0, separate processes and images and pods run to drive the kubernetes
+API and pod managers. The control plane services run as static pods so they can
+manage normal workloads or processes the same way that they manage disaster
+recovery. Aside from the core control plane components, other services run as 
+normal pods on the cluster, managed by regular Kubernetes constructs. Unlike in the past
+where the `kubelet` could be running as containerized or non-containerized, the `kubelet`
+always runs as a `systemd` process.
+
+
+[id='cluster-version-operator-{context}']
+== The cluster version Operator
+
+The cluster version Operator orchestrates all things.
+
+{product-title} 4.0 introduces several new components that support the cluster
+version Operator, including Cincinnati and Telemetry.
+
+Cincinnati is the hosted service that provides over the air updates to both 
+{product-title} and RHCOS. It provides a graph, or diagram that contain
+_vertices_ and the _edges_ that connect them, of component Operators. The edges
+in the graph show which versions you can safely upgrade to. The cluster version
+Operator checks with Cincinnati and determines valid upgrades and upgrade paths
+based on current component versions and information in the graph. If you
+configure it to do so, Cincinnati sends the release artifacts that it needs to
+perform the upgrade to your image registry, and the cluster version Operator
+upgrades your cluster. By accepting automatic updates, you can automatically
+keep your cluster up to date with the most recent compatible components.
+
+To allow Cincinnati to provide only compatible updates, a release verification
```

Comment:
Don’t call it Cincinnati.  Say it’s the OpenShift update service.  Cincinnati is more of a code name.